src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const result = days * 40;
```

Comment:
I think result is not the best name for this variable - in rturn statement you are not return result directly, but modified value of it.
What is "40" - maybe it would be better to extract it to named constant and move it above function body.